Components/Queues/Drivers/QueueDriverInterface.php

```diff
@@ -108,5 +108,5 @@ public function doDestroy();
     /**
      * @return int The time it took in ms to de-queue and deserialize the message
      */
-    public function getDequeueingTimeMs();
+    public function getDequeueTimeMs();
```

Comment:
Which is the reason of changing the interface signature? This provokes a breaking change in the bundle because this method is public and can be used for anybody

Reply:
This change was rollbackED